tests/pipelines/test_predict_step.py

```diff
@@ -43,12 +46,21 @@ def predict_input(tmp_pipeline_exec_path: Path):
     )
     proprocessing_step_output_dir.mkdir(parents=True)
     X, _ = load_diabetes(as_frame=True, return_X_y=True)
-    X.to_parquet(proprocessing_step_output_dir.joinpath(_CLEANED_OUTPUT_FILE_NAME))
-
-
-def test_predict_step_run(tmp_pipeline_root_path: Path, tmp_pipeline_exec_path: Path):
-    run_id, _ = train_and_log_model()
-    model_uri = "runs:/{run_id}/{artifact_path}".format(run_id=run_id, artifact_path="train/model")
+    X.to_parquet(proprocessing_step_output_dir.joinpath(_PREPROCESSED_OUTPUT_FILE_NAME))
```

Comment:
small nit: I would make this name more descriptive than `X`

Reply:
Hmmm, would you be comfortable keeping it like this? I'm following existing naming patterns in the helper functions file.

Reply:
Merging for now, let me know if you feel strongly about this. :)